medcat/config_meta_cat.py

```diff
@@ -27,8 +27,21 @@ class General(MixingConfig, BaseModel):
     """What category is this meta_cat model predicting/training.
 
     NB! For these changes to take effect, the pipe would need to be recreated."""
+    alternative_category_names: List = []
+    """List that stores the variations of possible category names
+    Example: For Experiencer, the alternate name is Subject
+    alternative_category_names: ['Experiencer','Subject']
+
+    In the case that one specified in 'category_name' parameter does not match the data, this ensures no error is raised and it is automatically mapped
+    """
     category_value2id: Dict = {}
     """Map from category values to ID, if empty it will be autocalculated during training"""
+    alternative_class_names: List[List] = [[]]
+    """Map that stores the variations of possible class names for the given category (task)
```

Comment:
Please change to `List of lists that stores...`, and again specify that the final output of the model will be the class name label specified in the the config  `<property>`